packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -164,6 +168,12 @@ class UnreferencedStateTracker {
     }
 }
 
+/**
+ * This sets the default session expiry. We have session expiry for GC purposes. Once the session expiry is
+ * set, we should not be able to change this value. Please look at TODO: link to GC readme.
+ */
+ export const defaultSessionExpiryMs = 30 * 24 * 60 * 60 * 1000; // 30 days
```

Comment:
Is this temporary? Or hardcoded forever?
I do not see it used - can we delete it until it actually is used such that we can assess correctness / impact for long term?

Reply:
It is hardcoded. We are not using it to expire sessions because we need to communicate with our partners before we enable session expiry. So yes, we can remove this for now and add it back when we enable it.

Reply:
Here's the PR for this change https://github.com/microsoft/FluidFramework/pull/8700